torchrl/objectives/ppo.py

```diff
@@ -664,6 +664,8 @@ class ClipPPOLoss(PPOLoss):
             ``"none"`` | ``"mean"`` | ``"sum"``. ``"none"``: no reduction will be applied,
             ``"mean"``: the sum of the output will be divided by the number of
             elements in the output, ``"sum"``: the output will be summed. Default: ``"mean"``.
+        clip_value_loss (bool, optional): if ``True``, the value loss will be clipped with respect to the
```

Comment:
I would have this
- not provided / False = no clipping
- True = same clipping as log ratio
- number = clip with that number 

That way you can reuse the same semantic for other losses